spec/spec_helper_local.rb

```diff
@@ -155,7 +155,7 @@ def param(type, title, param)
 # See https://github.com/voxpupuli/voxpupuli-test/blob/master/lib/voxpupuli/test/facts.rb
 add_custom_fact :service_provider, ->(_os, facts) do
```

Comment:
A quick look at the `service_provider` fact:
```console
$ rg service_provider manifests/
manifests/server/service.pp
8:    service_provider => $postgresql::server::service_provider,

manifests/server/instance/service.pp
7:# @param service_provider Overrides the default PostgreSQL service provider.
25:  Optional[String[1]]                          $service_provider = $postgresql::server::service_provider,
40:      provider  => $service_provider,

manifests/globals.pp
22:# @param service_provider Overrides the default PostgreSQL service provider.
119:  Optional[String[1]] $service_provider            = undef,

manifests/server.pp
19:# @param service_provider Overrides the default PostgreSQL service provider.
131:  Optional[String[1]]                                $service_provider             = $postgresql::params::service_provider,

manifests/params.pp
20:  $service_provider             = $postgresql::globals::service_provider

manifests/server/instance/config.pp
247:  if $facts['os']['family'] in ['RedHat', 'Gentoo'] and $facts['service_provider'] == 'systemd' {
```

The last one probably can't be simplified because Gentoo also supports non-systemd (like openrc).

Reply:
yeah. Ideally the whole file would go away :(